### DISCUSSION TITLE
See older anomalies

### DISCUSSION THREAD

**Discussion by sdepablos** [cuebook/CueObserve#139] [Ideas]
Sometimes, especially when a new anomaly definition is created, checking old anomalies is needed (and executing the RCA), but right now is not possible, only the most recent anomaly is available.

**Comment by sachinkbansal** [comment]
@sdepablos trying to understand the use case here.

Say you have the below anomaly card where the latest anomaly is on 2021-08-15. There are 4 older anomalies as well (in red dot).

![image](https://user-images.githubusercontent.com/40480714/133975306-e5975385-c50f-4a9b-a4c8-095a64305f2a.png)

Questions:
1. Do you want to see anomalies older than 2021-07-03?
2. Do you want to execute RCA on an older anomaly?

**Reply by sdepablos** [reply]
Both: being able to see older ones, and being able to execute RCA on all of them. For #2 my workaround right now is to add a new dataset with no data after a given anomaly date, so it's the last one and I can RCA it.

**Reply by sachinkbansal** [reply]
created issues https://github.com/cuebook/CueObserve/issues/141 and https://github.com/cuebook/CueObserve/issues/142

**Reply by sdepablos** [reply]
:+1:
